pkg/controllers/streaming/processor_controller.go

```diff
@@ -532,6 +533,15 @@ func (r *ProcessorReconciler) constructDeploymentForProcessor(processor *streami
 		ImagePullPolicy: v1.PullIfNotPresent,
 		Env:             environmentVariables,
 		VolumeMounts:    volumeMounts,
+		ReadinessProbe: &v1.Probe{
+			Handler: v1.Handler{
+				TCPSocket: &v1.TCPSocketAction{
+					Port: intstr.IntOrString{
+						IntVal: 8081,
```

Comment:
it's a bit odd to have a container's probe reference a port claimed by a different container in the pod. Each container should have it's own probes

Reply:
duh! fixed.